### PR TITLE
[WIP] PID tuning v2

### PR DESCRIPTION
**Pull request by dagar**
@anton-matosov's PID tuning 

Original https://github.com/mavlink/qgroundcontrol/pull/4458

### CODE REVIEW

**Comment by dagar** [comment]
I'll resolve the merge conflicts, then can we discuss this @DonLakeFlyer?

**Comment by DonLakeFlyer** [comment]
Yeah, I'm out of town today-sunday. Won't be till next week.

**Comment by dagar** [comment]
Not urgent, but I think having semi-accessible tuning screens is quite important.

**Comment by DonLakeFlyer** [comment]
The other really big mess is log replay which is pretty broken right now. That also hurts tuning I think.

**Comment by dagar** [comment]
Log replay is far more approachable for review in general to regular users. Even technical users initially struggle reviewing autopilot logs because they don't even know where to begin with the different fields.

**Comment by dagar** [comment]
https://github.com/mavlink/qgroundcontrol/issues/2878

**Comment by jaxxzer** [comment]
@dagar can you post a picture of what this looks like?

**Comment by dagar** [comment]
I broke it during the rebase, let me try again.

**Comment by anton-matosov** [comment]
Hi @dagar and @DonLakeFlyer, it's been a while since I was here last time =) 
Are there any updates on these changes?

**Comment by DonLakeFlyer** [comment]
I hope to get to pid tuning in 3.4

**Comment by DonLakeFlyer** [comment]
Replaced by the new pid tuning stuff wip.
